app/services/user.py

```diff
@@ -80,7 +81,10 @@ def __init__(self, **attrs):
     def create(self) -> User:
         try:
             new_user = User(
-                email=self.email, confirmed=False, tier=self.get_user_tier()
+                email=self.email,
+                confirmed=False,
+                tier=self.get_user_tier(),
+                uuid=self.get_user_uuid(),
```

Comment:
This helper seems indirect since you're just generating the uuid.  Why not just `str(uuid.uuid4())`?